src/components/Profile/Collected.tsx

```diff
@@ -72,9 +70,12 @@ const Collected = () => {
       )}
       <SimpleGrid
         ref={sentryRef}
-        minChildWidth={displaySize}
-        w="full"
-        spacing="4"
+        maxW="1050px"
+        w="100%"
+        mx="auto"
+        columns={[1, 2, 3]}
```

Comment:
We might want to keep this responsive object uniform across the codebase. 
Currently we might want to do it this way:  `{{base: 1, md: 2, xl: 3}}`. This was we keep it uniform and increase readability.

Reply:
```suggestion
        columns={{base: 1, md: 2, lg: 3}}
```

@codiejay remember you can suggest if its a one line change so the PR creator can commit the change with one click 👍🏻

Reply:
Ah thanks @kesar  - will keep that in mind!!